tools/ci-ts/test-helpers/common.ts

```diff
@@ -67,57 +78,61 @@ export function getArgs<T extends string>(keys: T[]): { [K in T]: string } {
   }, {} as { [K in T]: string })
 }
 
-/**
- * Registers a global promise handler that will exit the currently
- * running process if an unhandled promise rejection is caught
- */
-export function registerPromiseHandler() {
-  process.on('unhandledRejection', e => {
-    console.error(e)
-    console.error(chalk.red('Exiting due to promise rejection'))
-    process.exit(1)
+export async function wait(ms: number) {
+  return new Promise(res => {
+    setTimeout(res, ms)
   })
 }
 
-interface DeployFactory {
-  new (signer?: ethers.Signer): ethers.ContractFactory
-}
-
-interface DeployContractArgs<T extends DeployFactory> {
-  Factory: T
-  name: string
-  signer: ethers.Signer
-}
-
-export async function deployContract<T extends DeployFactory>(
-  { Factory, name, signer }: DeployContractArgs<T>,
-  ...deployArgs: Parameters<InstanceType<T>['deploy']>
-): Promise<ReturnType<InstanceType<T>['deploy']>> {
-  const contractFactory = new Factory(signer)
-  const contract = await contractFactory.deploy(...deployArgs)
-  await contract.deployed()
-  console.log(`Deployed ${name} at: ${contract.address}`)
-
-  return contract as any
+/**
+ * changePriceFeed makes a patch request to the external adapter in tools/external-adapter
+ * and changes the value reported
+ * @param adapter the URL of the external adapter
+ * @param value the value to set on the adapter
+ */
+export async function changePriceFeed(adapter: string, value: number) {
```

Comment:
This function is using DOM specific global functions, like `URL` and `fetch`, we should use either node specific or isomorphic functions for this. Then, in `jest.config.js` we should be targeting `node` env, rather than the default `browser` env.

Reply:
Okay tell me if this addresses everything:
* imported `isomorphic-unfetch` in common.ts (I wanted to replicate what we had done in other places rather than add a new dependency for http)
* jest.config.js has `testEnvironment` set to "node"
* Looks like node supports the [URL api](https://nodejs.org/api/url.html)

Reply:
Sounds good, I would just make sure that you're importing `url` rather than depending on a globally available one.

Reply:
Looks like as of node v10, URL is available on the global object and the `url` module has different functionality.

https://nodejs.org/api/url.html#url_class_url

Reply:
Oh awesome, TIL. Looks good then!